service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java

```diff
@@ -38,17 +38,16 @@
     private final RepresentativeNotificationService representativeNotificationService;
     private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
     private final GeneratedOrderService generatedOrderService;
-    private final FeatureToggleService featureToggleService;
 
     @EventListener
     public void notifyParties(final GeneratedOrderEvent orderEvent) {
         final CaseData caseData = orderEvent.getCaseData();
-        final byte[] documentContents = orderEvent.getDocumentContents();
+        final DocumentReference orderDocument = orderEvent.getOrderDocument();
 
-        issuedOrderAdminNotificationHandler.notifyAdmin(caseData, orderEvent.getDocumentContents(), GENERATED_ORDER);
+        issuedOrderAdminNotificationHandler.notifyAdmin(caseData, orderDocument.getBinaryUrl(), GENERATED_ORDER);
```

Comment:
inconsistency, it would be better to pass entire orderDocument, same as in sendNotificationToEmailServedRepresentatives(caseData, orderDocument);